package.json

```diff
@@ -253,6 +253,39 @@
         ],
         "visible": true
       },
+      {
+        "permissionName": "ui-users.settings.customfields.view",
+        "displayName": "Settings (Users): Can view custom fields",
+        "subPermissions": [
+          "customfields.collection.get",
+          "customfields.item.get",
+          "custom.fields.item.stats.get",
+          "settings.users.enabled"
+        ],
+        "visible": true
+      },
+      {
+        "permissionName": "ui-users.settings.customfields.edit",
+        "displayName": "Settings (Users): Can create, edit, and view custom fields",
+        "subPermissions": [
+          "ui-users.settings.customfields.view",
+          "custom.fields.collection.put",
+          "customfields.item.post",
+          "customfields.item.put"
+        ],
```

Comment:
We also need to add `configuration.entries.item.post` and `configuration.entries.item.put` to the `customfields.edit` subPermissions since editing custom fields also implies the ability to view and edit the custom fields accordion label.

Reply:
Do we also need to add `configuration.etries.item.delete` to `customfields.delete` subPermissions?